src/thrift/person.thrift

```diff
@@ -0,0 +1,7 @@
+namespace py thrift_example.gen
```

Comment:
Ditto on a comment making the connection with this line -> where it gets generated. If you take the above recommendation, I think it'd be good to have here too. Duplication can be a good thing with documentation.

Reply:
Done.